package.json

```diff
@@ -45,6 +45,7 @@
     "cssnano": "^4.1.10",
     "d3-array": "^2.8.0",
     "d3-collection": "^1.0.7",
+    "escape-string-regexp": "^4.0.0",
```

Comment:
Wow! This is the whole library!

```js
string
  .replace(/[|\\{}()[\]^$+*?.]/g, '\\$&')
  .replace(/-/g, '\\x2d')
```

I think writing it yourself could save you one `if` ~and one `import`~ but it's shared in 2 places.